src/main/java/ch/admin/bag/covidcertificate/sdk/core/data/AcceptanceCriteriasConstants.kt

```diff
@@ -13,10 +13,12 @@ package ch.admin.bag.covidcertificate.sdk.core.data
 object AcceptanceCriteriasConstants {
 
 	const val NEGATIVE_CODE: String = "260415000"
+	const val POSITIVE_CODE: String = "260373001"
 	const val TARGET_DISEASE = "840539006"
 }
 
 enum class TestType(val code: String) {
 	RAT("LP217198-3"),
-	PCR("LP6464-4")
+	PCR("LP6464-4"),
+	SERO_POSITIV("94504-8")
```

Comment:
Does it make sense to call the TestType SERO_POSITIVE? should it not be only SERO? positive is the result, but could also be negative... Otherwise you would also need to call it RAT_NEGATIVE AND PCR_NEGATIVE